Comment on timestamp literals (#2467)
duckdb/duckdb-web: docs/sql/data_types/timestamp.md

| `TIMESTAMP_S`  |                            | timestamp with second precision (ignores time zone)      |
| `TIMESTAMPTZ`  | `TIMESTAMP WITH TIME ZONE` | timestamp (uses time zone)                               |

> A timestamp specifies a combination of [`DATE`](date) (year, month, day) and a [`TIME`](time) (hour, minute, second, microsecond). Timestamps can be created using the `TIMESTAMP` keyword, where the data must be formatted according to the ISO 8601 format (`YYYY-MM-DD hh:mm:ss[.zzzzzz][+-TT[:tt]]`). Decimal places beyond the targeted sub-second precision are ignored. Additionally, the `TIMESTAMP_NS` keyword ignores decimal places beyond _microseconds_ despite the underlying type being able to hold nanoseconds when created via ingestion of, for example, parquet files.

Reviewer: ```suggestion
A timestamp specifies a combination of [`DATE`](date) (year, month, day) and a [`TIME`](time) (hour, minute, second, microsecond). Timestamps can be created using the `TIMESTAMP` keyword, where the data must be formatted according to the ISO 8601 format (`YYYY-MM-DD hh:mm:ss[.zzzzzz][+-TT[:tt]]`). Decimal places beyond the targeted sub-second precision are ignored. Additionally, the `TIMESTAMP_NS` keyword ignores decimal places beyond _microseconds_. Note that the underlying type is able to hold nanoseconds when created e.g. via ingestion the ingestion of Parquet files.
```
